missions/LIL_Missions.txt

```diff
@@ -289,27 +271,10 @@ LIL_diplo_missions_2 = {
 			}
 			if = {
 				limit = {
-					reverse_has_opinion = {
-						who = CRD
-						value = 100
-					}
+					exists = CRD
 				}
 				country_event = { id = flavor_lil.1 }
 			}
-			else = {
-				limit = {
-					OR = {
-						is_rival = CRD
-						is_enemy = CRD
-					}
-				}
-				define_general = {
-					shock = 3
-					fire = 3
-					manuever = 4
-					siege = 2
-				}
-			}
```

Comment:
Why removing stuff with CRD?

Reply:
i guess i have an older version of some of the mission files, because i really don't remember touching or changing many of the issues mentioned here